Backend/tests/test_post_service.py

```diff
@@ -0,0 +1,196 @@
+import pytest
```

Comment:
Instead of creating vars to each dependency you inject, 
you can use pytest fixtures - 

for example (credit to chat gpt) 
```

import pytest

@pytest.fixture
def setup_data():
    data = {"key": "value"}
    # Setup actions can go here
    yield data  # The fixture function can yield the data to provide it to the test
    # Teardown actions (optional) can go here
    
    

def test_example(setup_data):
    assert "key" in setup_data
    assert setup_data["key"] == "value"
```

Reply:
Check how to use it, try avoiding creating the same properties duplicated